pytorch_lightning/trainer/trainer.py

```diff
@@ -713,42 +713,86 @@ def run_sanity_check(self, ref_model):
             self.on_sanity_check_end()
             self.running_sanity_check = False
 
-    def test(
+    def validate(
         self,
         model: Optional[LightningModule] = None,
-        test_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
+        val_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
```

Comment:
Does the `limit_val_batches` trainer param apply here?

Reply:
Yes!